docs/metrics/customizing-the-sdk/README.md

```diff
@@ -372,6 +372,9 @@ OpenTelemetry SDK comes with the following `ExemplarFilters` (defined on
 The `SetExemplarFilter` extension method on `MeterProviderBuilder` can be used
 to set the desired `ExemplarFilterType` and enable `Exemplar` collection:
 
+> [!NOTE]
+> The `SetExemplarFilter` API was added in the `1.9.0` release.
```

Comment:
1.9.0 is not released, right?

Reply:
Correct. I figured we would put it out soon (alpha/beta) after this lands so it wouldn't be a big deal to mention it here a bit early. What you think?